std_msgs/Header header
duration t1
duration t2
duration t3
